wp-content/themes/core/assets/css/src/theme/forms/_variables.pcss

```diff
@@ -76,21 +53,22 @@
 	--form-placeholder-text-transform: var(--form-text-transform);
 
 	/* -----------------------------------------------------------------------------
- * Controls
- * ----------------------------------------------------------------------------- */
+	 * Controls
+	 * ----------------------------------------------------------------------------- */
 
 	--form-control-bgd-color: var(--color-white);
-	--form-control-border-color: var(--form-border-color);
-	--form-control-border-color-focus: var(--form-border-color-focus);
-	--form-control-border-radius: var(--form-border-radius);
-	--form-control-border-size: var(--form-border-size);
-	--form-control-border-size-focus: var(--form-border-size-focus);
+	--form-control-border-color: var(--color-neutral-20);
+	--form-control-border-color-focus: var(--color-neutral-90);
+	--form-control-border-radius: 0;
+	--form-control-border-size: 1px;
+	--form-control-border-size-focus: var(--form-control-border-size);
```

Comment:
I think my preference would be to keep the global set of `--form-border-`vars up top and nix these control & field vars in favor of using those. Thoughts? It's nice to have sets of vars segmented into their groupings for both organization but clarity too.

Reply:
I was 50/50 on keeping them just as `form-border--` up top, or adding them with the controls. Since they were used for controls, I figured it was logical to add them w/ controls BUT it's less to type if we just keep them as `form-border--*` up top, I'm in favor of that